guides/v2.2/ui_comp_guide/bk-ui_comps.md

```diff
@@ -70,7 +70,7 @@ UI component is a combination of:
 2. **JavaScript** class inherited from one of the Magento JavaScript framework UI components base classes (such as [UIElement]({{ page.baseurl }}/ui_comp_guide/concepts/ui_comp_uielement_concept.html), [UIClass]({{ page.baseurl }}/ui_comp_guide/concepts/ui_comp_uiclass_concept.html) or [UICollection]({{ page.baseurl }}/ui_comp_guide/concepts/ui_comp_uicollection_concept.html)).
 
 
-3. **Related template(s)**
+3. **Related template(s)** using KnockoutJs bindings
```

Comment:
Please, remove this clarification. The existing list is about "what", but not "how". So, let's keep it simple.
Thank you.

Reply:
@rogyar removed clarification. make it simple